parse.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  /**
+   * @description Parses env file data
+   * @param {string | Buffer} envStr
+   * @returns {{
+   *   key: string,
+   *   value: string,
+   *   isEnvVar: boolean,
+   * }[]}
+   */
+  parse(envStr = '') {
+    const keyValuePattern = /^\s*([\w.-]+)\s*=\s*("[^"]*"|'[^']*'|[^#]*)?(\s*|\s*#.*)?$/;
```

Comment:
This RegExp might look scary but here is a reference from [dotenv](https://github.com/motdotla/dotenv/blob/master/lib/main.js#L10)